{"name": "test_fixture_zcc4[cc_calculator_shuju4]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 8 == 2", "trace": "self = <cc_work.test_fixture_zcc.TestCal_cc_new object at 0x00000000039E2C50>, cc_calculator_func = <cc_work.calculator_cc.Calculator object at 0x00000000039E27B8>, cc_calculator_shuju = [4, 2, 2]\n\n    @pytest.mark.run(order=3)\n    def test_fixture_zcc4(self, cc_calculator_func, cc_calculator_shuju):\n        result = cc_calculator_func.mul(cc_calculator_shuju[0], cc_calculator_shuju[1])\n        print('这是测试用例4')\n        # print(result)\n>       assert result == cc_calculator_shuju[2]\nE       assert 8 == 2\n\ntest_fixture_zcc.py:46: AssertionError"}, "attachments": [{"name": "stdout", "source": "10ff31c6-e60e-492c-b596-28412e2b3711-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "cc_calculator_shuju", "value": "[4, 2, 2]"}], "start": 1609384405979, "stop": 1609384405982, "uuid": "e6094343-fb23-4645-86c8-4835fd7c161f", "historyId": "7f43400bd6053ab7b89e4e86f686bfe2", "testCaseId": "000433c5f8a90c3b0946db3d61bfc89d", "fullName": "test_fixture_zcc.TestCal_cc_new#test_fixture_zcc4", "labels": [{"name": "suite", "value": "test_fixture_zcc"}, {"name": "subSuite", "value": "TestCal_cc_new"}, {"name": "host", "value": "admin-PC"}, {"name": "thread", "value": "12648-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_fixture_zcc"}]}